app/models/chargeback_rate_detail.rb

```diff
@@ -175,11 +176,15 @@ def contiguous_tiers?
         error = true if tier.ends_with_infinity?
       end
 
-      break if error
+      ambiguous_error = tier.start == tier.finish
```

Comment:
@lpichler As we discussed, please move this logic into a method named `ambiguous_tiers?`